lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -54,6 +55,12 @@ function getNodesAndTimingByUrl(nodeTimings) {
     const nodeTiming = nodeTimings.get(node);
     if (!nodeTiming) return;
 
+    if (usesAMP && node.record.resourceType === 'Stylesheet'
```

Comment:
We'll definitely need a beefy comment here on why this is fair and not totally cheating/playing nice for AMP.